src/items/FileItem.stories.tsx

```diff
@@ -0,0 +1,43 @@
+// TODO
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import React from 'react';
+
+import { ItemType } from '@graasp/sdk';
+
+import { MIME_TYPES } from '../constants';
+import { ImmutableItem } from '../types';
+import { TABLE_CATEGORIES } from '../utils/storybook';
+import FileItem from './FileItem';
+
+export default {
+  title: 'Items/FileItem',
+  component: FileItem,
+
+  argTypes: {
+    sx: {
+      table: {
+        category: TABLE_CATEGORIES.MUI,
+      },
+    },
+  },
+} as ComponentMeta<typeof FileItem>;
+
+const Template: ComponentStory<typeof FileItem> = (args) => (
+  <FileItem {...args} />
+);
+
+export const DefaultImage = Template.bind({});
+DefaultImage.args = {
```

Comment:
Can't really see anything in this story:

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/1707188/197232121-629856c7-0613-43d1-9f5e-338807f19176.png">